lib/cosmos/io/json_drb.rb

```diff
@@ -132,6 +132,8 @@ def start_service(hostname = nil, port = nil, object = nil)
               :Host   => hostname,
               :Port   => port,
               :Silent => true,
+              :Verbose => false,
+              :Threads => "0:#{max_threads}",
```

Comment:
Where are these parameters documented?

Reply:
I found them in the puma source.  Don't know where else?